src/screen/ConnectionsList/Connection/index.tsx

```diff
@@ -59,55 +74,140 @@ const Connection: React.FC<IConnectionProps> = ({ connection }) => {
         setIsConnecting(false)
       }
     }
-  }, [])
+  }, [isConnected, connection, setCurrentConnection, setCurrentDatabase])
+
+  const handleDisconnect = useCallback(async () => {
+    setCurrentConnection(undefined)
+    setCurrentDatabase(undefined)
```

Comment:
I think at this moment we should also disconnect the Redis connection made here https://github.com/diego3g/rocketredis/blob/master/src/services/RedisConnection.ts, what do you think?

Reply:
Good call. I thought about exporting a `terminateConnection` function. What do you think?

Also, should this function be allowed to receive `undefined`?